src/import-export/azext_import_export/generated/_help.py

```diff
@@ -0,0 +1,85 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+# pylint: disable=line-too-long
+# pylint: disable=too-many-lines
+
+from knack.help_files import helps
+
+
+helps['import-export'] = """
+    type: group
+    short-summary: Manage Import Export
+"""
+
+helps['import-export list'] = """
+    type: command
+    short-summary: Returns all active and completed jobs in a subscription.
+    examples:
+      - name: List jobs in a resource group
+        text: |-
+               az import-export list --resource-group "myResourceGroup"
+      - name: List jobs in current subscription
+        text: |-
+               az import-export list
+"""
+
+helps['import-export show'] = """
+    type: command
+    short-summary: Gets information about an existing job.
+    examples:
+      - name: Get job
+        text: |-
+               az import-export show --resource-group "myResourceGroup" --name "myJob"
+"""
+
+helps['import-export create'] = """
+    type: command
+    short-summary: Creates a new job or updates an existing job in the specified subscription.
+    examples:
+      - name: Create an import job
+        text: |-
+               az import-export create --resource-group "myResourceGroup" --name "myJob"
+               --location "West US" --backup-drive-manifest true --diagnostics-path "waimportexport"
+               --drive-list bit-locker-key=238810-662376-448998-450120-652806-203390-606320-483076
```

Comment:
drive-list is an array right

Reply:
yes, we can specify multi --dirve-list here.